beacon-chain/db/block_header.go

```diff
@@ -0,0 +1,121 @@
+package db
+
+import (
+	"bytes"
+
+	"github.com/boltdb/bolt"
+	"github.com/gogo/protobuf/proto"
+	"github.com/pkg/errors"
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/shared/featureconfig"
+)
+
+func createBlockHeader(enc []byte) (*ethpb.BeaconBlockHeader, error) {
```

Comment:
im not sure if this complies with the new db standard or this is needed for the new DB, perhaps we should talk about this being a completely different service since this should not be part of beacon node

Reply:
we certainly should talk about it.
I can see a situation where there is a competing chain with different network parameters that we have to listen for slashing purposes. i can for now just add the relevant filters for my needs based on blocks